doc/api/esm.md

```diff
@@ -241,13 +241,27 @@ To include an ES module into CommonJS, use [`import()`][].
 
 ### `import` statements
 
-An `import` statement can reference an ES module or a CommonJS module. Other
-file types such as JSON or native modules are not supported. For those, use
-[`module.createRequire()`][].
-
+An `import` statement can reference an ES module or a CommonJS module.
 `import` statements are permitted only in ES modules. For similar functionality
 in CommonJS, see [`import()`][].
 
+Additional experimental flags are available for importing
+[Wasm modules](#esm_experimental_wasm_modules) or
+[JSON modules](#esm_experimental_json_modules). For importing native modules or
+JSON modules unflagged, see [`module.createRequire()`][].
+
+Only the “default export” is supported for importing CommonJS files or packages:
+
+<!-- eslint-disable no-duplicate-imports -->
+```js
+import packageMain from 'commonjs-package'; // Works
+
+import { method } from 'commonjs-package'; // Errors
+```
+
+Support is provided for detecting named exports on CommonJS modules that are
+transpiled from ES modules and expose the `__esModule` "flag" export.
```

Comment:
```suggestion
```

I suggest to either remove this section, or write a small sentence that links to the section below about "CommonJS Imports"

Reply:
Thanks, I missed that. Added a short description and a link. Let me know how that looks.